cmd/namespace/create.go

```diff
@@ -60,10 +61,11 @@ func Create(ctx context.Context) *cobra.Command {
 			analytics.TrackCreateNamespace(err == nil)
 			return err
 		},
-		Args: utils.ExactArgsAccepted(1, ""),
+		Args: utils.MaximumNArgsAccepted(2, ""),
 	}
 
 	options.Members = cmd.Flags().StringArrayP("members", "m", []string{}, "members of the namespace, it can the username or email")
+	cmd.Flags().BoolVarP(&options.SetCurrent, "set current", "u", false, "use to set ns as default ns while creating")
```

Comment:
@pchico83 @rlamana Can you help with the naming of this new flag? 

We need a way of preserving the current approach (changing to the new namespace created if the user doesn't set a flag). If we set it default to true to set it to false the user would do something like `okteto namespace create --set-current=false`. I guess we could do something like we do for the `--no-bash` and name it something like `--no-switch` so the default value is `false` keeping the current behavior untouched and the new behavior would look like `okteto namespace create --no-switch`. WDYT?

Reply:
or more simply we can use `--use`. If `--use` flag is specified, we can set the newly created namespace as the current namespace. And if not, the namespace will be created but not be set as current namespace.

Reply:
It's explain [here](https://github.com/okteto/okteto/issues/3579#issuecomment-1524709937).
The flag is `--use`, to match the `okteto ctx use` command. The default value is `true` to avoid breaking changes and keep the current behavior. And you can disable it with `okteto ns create --use=false`

Reply:
@jLopezbarb I'll work on adding a new flag name. Could you help me on how to set the flag as `--use`. The shorthand is exceeding the length if i take more than 1 character.

Reply:
Let's only use the longer syntax then:
```suggestion
	cmd.Flags().BoolVarP(&options.SetCurrent, "use", "", true, "use to set ns as default ns while creating")
```